web/templates/phrase/show.html.eex

```diff
@@ -0,0 +1,18 @@
+<h2>Show phrase</h2>
+
+<ul>
+
+  <li>
+    <strong>Message:</strong>
+    <%= @phrase.message %>
+  </li>
+
+  <li>
+    <strong>Has token:</strong>
+    <%= @phrase.has_token %>
```

Comment:
Was it needed for the edit page? I remember @daveshah and I hitting some issues when removing some pages. If there's no need for it, I'd remove it